web/src/views/CreateDandisetView/CreateDandisetView.vue

```diff
@@ -53,16 +56,21 @@ export default {
       return !(this.name && this.description);
     },
   },
+  mounted() {
```

Comment:
Any particular reason to use the `mounted` lifecycle hook instead of `created`?

Reply:
Agreed. Unless it directly relies on the DOM, I'd default to always using `created`?

Reply:
I had a vague idea of `mounted` being more "reliable", but I'm not sure where I got that idea from. `created` is the better option here, thanks